app/components/utils/search-legend/template.hbs

```diff
@@ -0,0 +1,34 @@
+<table class="vl-data-table tr-hover">
+  <thead>
+    <tr>
+      <th class="vl-data-table-col-2 vl-data-table__header-title">
+        {{t "search-legend-header-example"}}
+      </th>
+      <th class="vl-data-table-col-9 vl-data-table__header-title">
+        {{t "search-legend-header-explanation"}}
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td class="vl-data-table-col-1">{{t "search-legend-and-example"}}</td>
+      <td class="vl-data-table-col-12">{{t "search-legend-and-explanation"}}</td>
+    </tr>
+    <tr>
```

Comment:
Gezien probelmen met `|` in searches, zou je deze (hbs-gewijs, eg `{{!-- --}})`) kunnen uitcommenten met wat uitleg? thx :-)

Reply:
Dat kan ik zeker doen!